_data/team.yml

```diff
@@ -105,3 +105,36 @@ fellows:
   link: https://twitter.com/glozow
   image: assets/images/gloria-zhao.jpg
   alt: Picture of Gloria Zhao
+
+grantees:
+- name: Jesse Posner
+  description: >-
+    Jesse’s funding supports his work on adaptor signatures and an
+    implementation of the FROST threshold signature scheme. These technologies
+    bring additional security, privacy and functionality to multisignature and
+    second layer protocols.
+  social: "@jesseposner"
+  link: https://twitter.com/jesseposner
+  image: assets/images/jesse-posner.jpg
+  alt: Picture of Jesse Posner
+
+- name: Alekos Filini
+  description: >-
+    Alekos’s funding supports his work as maintainer of the Bitcoin Developer
```

Comment:
Is "Developer" actually in the name?  Looking at the website and the README for the repository, I only see "Dev".  The LN kit is "Lightning *Development* Kit", so I'd have expected this to use basically the same name.

If you change this, it also needs to be updated in the blog post.

Reply:
:facepalm: my mistake. I've changed it to "Bitcoin Dev Kit" in both places.